django-verdant/rcasite/settings/production.py

```diff
@@ -164,107 +144,28 @@
 
 
 # Elasticsearch
-
-if 'ELASTICSEARCH_URL' in env:
+if 'SEARCHBOX_URL' in env:
```

Comment:
Probably should remain `ELASTICSEARCH_URL` since that's the API we're expecting on the other end. Doesn't matter who hosts it